pkg/resource/predicates.go

```diff
@@ -72,3 +72,14 @@ func HasProvisioner(c client.Client, cr ClassReferencer, provisioner string) boo
 
 	return strings.EqualFold(cs.Provisioner, provisioner)
 }
+
+// NoClassRef accepts ResourceClaims that do not reference a specific ResourceClass
+func NoClassRef() PredicateFn {
```

Comment:
Nitpick, but we use the full `Reference` in most places in the codebase at the moment. I'd prefer to keep consistent.